charts/gatekeeper/templates/gatekeeper-manager-role-clusterrole.yaml

```diff
@@ -137,3 +137,17 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
only when flag is enabled

(this change should be in helmify and should modify manifest_staging/charts)